CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,75 @@
+# Contributor Covenant Code of Conduct
+
+# Our Pledge
```

Comment:
[nit] Only the title should use `#` all the rest should add one more (like there should only be one `h1` in a document, then the outline should use `h2`s...)